internal/controller/backup-service/reconciler.go

```diff
@@ -348,18 +348,30 @@ func (r *SingleBackupServiceReconciler) getBackupSvcDeployment() (*app.Deploymen
 }
 
 func (r *SingleBackupServiceReconciler) updateBackupSvcConfig() error {
+	if r.aeroBackupService.Status.Config.Raw == nil {
+		r.Log.Info("Skipping backup service config reload as status is empty")
```

Comment:
add backup service name and namespace in log message.

Reply:
That info is already added in the logger while initialising the reconciler